cloudformation/security-hq.template.yaml

```diff
@@ -54,6 +54,9 @@ Parameters:
   TopicARN:
     Type: String
     Description: ARN for Anghammarad's SNS topic
+  SNSTopicForAlerts:
```

Comment:
I think we can cloudform this but perhaps we'll end up supplying the topic subscriber email address as a parameter

Reply:
Indeed! It's actually easier than I expected